docs/simulating-webhooks.md

```diff
@@ -20,4 +20,4 @@ Note that `event-name` here is just the name of the event (like pull_request or
 
 For example, to simulate receiving the `pull_request.labeled` event, run:
 
-    $ node_modules/.bin/probot receive -e pull_request -p test/fixtures/pull_request.labeled.json ./index.js
+    $ node_modules/.bin/probot receive -e pull_request.labeled -p test/fixtures/pull_request.labeled.json ./index.js
```

Comment:
the `labeled` part is part of the payload, I don't think that change is correct. Did you test it?

Reply:
I tested it with `pull_request.merged`, but when I tried it now, it worked as `pull_reuqest`. Sorry, my mistake 🙇